Tools/CISupport/ews-build/steps.py

```diff
@@ -5930,6 +5931,96 @@ def getResultSummary(self):
         return {'step': configuration}
 
 
+class CheckOutLLVMProject(git.Git, AddToLogMixin):
+    name = 'checkout-llvm-project'
+    directory = 'llvm-project'
+    branch = 'webkit'
+    CHECKOUT_DELAY_AND_MAX_RETRIES_PAIR = (0, 2)
+    GIT_HASH_LENGTH = 40
```

Comment:
Right. I just took the value from the `Git` built-in BuildStep. I think it's also 40 for `CheckoutSource` and it's only really used to check the revision length after a successful checkout in `parseGotRevision` rather than actually truncating a hash.